pkg/controller/ps/controller.go

```diff
@@ -1552,6 +1552,17 @@ func (r *PerconaServerMySQLReconciler) createSSLByCertManager(ctx context.Contex
 		issuerKind = cr.Spec.TLS.IssuerConf.Kind
 		issuerName = cr.Spec.TLS.IssuerConf.Name
 		issuerGroup = cr.Spec.TLS.IssuerConf.Group
+
+		isr := &cm.Issuer{}
+		err := r.Get(ctx, types.NamespacedName{
+			Namespace: cr.Namespace,
+			Name:      issuerName,
+		}, isr)
+		if k8serrors.IsNotFound(err) {
+			log.FromContext(ctx).Error(err, "Can't find specified cert-manager issuer", "issuer", issuerName)
```

Comment:
do we need a separate log? if we wrap the error with this message, it'll be logged anyway

Reply:
@egegunes I was waiting for that question :) Yes, that is totally true, my idea was to make it even more obvious to the user (so does not have to go through the generic `reconcile` error, but I also think we don't need it necessarily.

Reply:
Removed